lib/core.js

```diff
@@ -126,6 +127,11 @@ function registerLanguage(name, syntax) {
   }
 }
 
+/* Get a list of all registered languages */
+function listLanguages() {
+  return languageNames.slice(0, languageNames.length)
```

Comment:
```suggestion
  return languageNames.concat()
```

As you’re making a clone here, I think `concat` is a cleaner way to do it!